src/test/rustdoc-ui/invalid-html-tags.stderr

```diff
@@ -0,0 +1,38 @@
+error: unclosed HTML tag `unknown`
+  --> $DIR/invalid-html-tags.rs:7:5
+   |
+LL | /// <unknown>
+   |     ^^^^^^^^
```

Comment:
Hmm, do you know why this doesn't also point to the closing `>` of the tag? Like this:
```suggestion
LL | /// <unknown>
   |     ^^^^^^^^^
```

Reply:
I think pulldown might be passing you the wrong span.

Reply:
It might be a whitespace or anything else in case you `<unknown style="1">`. So I preferred to only point to the start of the tag in case attributes were also present.